templates/resources/service_integration.md.tmpl

```diff
@@ -7,12 +7,20 @@ description: |-
 # {{.Name}} ({{.Type}})
 {{ .Description | trimspace }}
 
+-> Services integrations are not supported for services running on hobbyist plans.
+
 You can set up an integration between two Aiven services or an Aiven service and an external
 service. For example, you can send metrics from a Kafka service to an M3DB service,
 send metrics from an M3DB service to a Grafana service to show dashboards, and send logs from 
 any service to OpenSearch.
 
-**Services integrations are not supported for services running on hobbyist plans.**
+You can also use service integrations to enable and use the [disk autoscaler](https://aiven.io/docs/platform/howto/disk-autoscaler).
+
+~> **Warning**
+For services managed by Terraform, removing an autoscaler integration on services with `additional_disk_space` resets the service disk space to the service plan's disk size.
+If the integration is managed by Terraform but not the service, the disk space is not reset. To retain the additional disk space set the service's `additional_disk_space` value manually.
```

Comment:
I would swap the last two sentences. Wdyt?

Reply:
Good call! ✅